database/src/migrations/template_methodology_species_validations/picklist_variables/v0.1.ts

```diff
@@ -254,16 +254,50 @@ const SignAge = [
 
 const GPSDatum = [{ name: 'NAD83' }, { name: 'NAD27' }, { name: 'WGS84' }];
 
-export const basicNumericValidator = [
-  {
-    column_numeric_validator: {
-      name: '',
-      description: ''
+interface objectNameDescription {
+  name: string;
+  description: string;
+}
+
+interface objectName {
+  name: string;
+}
```

Comment:
I assume the idea is that description is optional?

In that case you can just add a questionmark, and then you only need 1 interface.
`description?: string;`